week-1/H-numbers/exercise.js

```diff
@@ -1 +1,13 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+
+/*Create two variables `numberOfStudents` and `numberOfMentors`
+Log a message that displays the total number of students and mentors*/
+
+var numberOfStudents = 50;
+var numberOfMentors = 60;
+var totalNUmberStudentsAndMentors = numberOfStudents + numberOfMentors;
+
+console.log('Number of students: ' + numberOfStudents);
+console.log('NUmber of mentors: ' + numberOfMentors);
+
+console.log('Totoal number of student and mentors: ' + totalNUmberStudentsAndMentors);
```

Comment:
Excellent - you've demonstrated how to use the different cases of the `+` operator and how to use existing variables when defining another variable.